package.json

```diff
@@ -13,15 +13,17 @@
     "LICENSES"
   ],
   "scripts": {
-    "lint": "npx eslint ."
+    "lint": "npx eslint .",
+    "test": "jest --ci --collectCoverage"
   },
   "devDependencies": {
-    "eslint": "^8"
+    "eslint": "^8",
+    "jest": "^29.7.0"
   },
   "peerDependencies": {
-    "@sap/cds": "7.9.4",
-    "@sap/cds-compiler" : "5.0.6",
     "@cap-js/asyncapi": "^1.0.0",
-    "@cap-js/openapi": "^1.0.2"
+    "@cap-js/openapi": "^1.0.2",
+    "@sap/cds": "7.9.4",
```

Comment:
Could we upgrade cds to 8 version, there is an issue existing: https://github.com/cap-js/ord/issues/50